### DISCUSSION TITLE
IfcSurfaceFeature assignment visualization in the Bonsai Interface

### DISCUSSION THREAD

**Discussion by fg55b** [IfcOpenShell/IfcOpenShell#6578] [General]
Greetings!

When working with IfcSurfaceFeatures associated to elements using the IfcRelAdheresToElement decomposition relationship, the BlenderBIM used to group the surface features inside the product decomposition, as shown below:

![image](https://github.com/user-attachments/assets/162fadc4-ff4b-400d-9a6a-1497f0bd1a2e)

In Bonsai, all surface features are grouped inside a "Unsorted" category, even though the same relationships are being used. The figure below shows the structure in Bonsai for the same file.

![image](https://github.com/user-attachments/assets/74d094c9-6f00-493f-bff6-ff33a42906ea)

In the "Object Information" scene, only the aggregate and nest decompositions are made explicit, so the surface features association can only be "visualized" in the debug inspector.

This behavior is expected or it should be adjusted in future versions? 

OBS: If I am not mistaken, the older versions of BlenderBIM automatically assigned both IfcRelAdheresToElement and IfcRelAggregates in the relationship between surface features and elements. Are they simultaneously necessary? To me, it seems redundant since both are subtypes of IfcRelDecomposes, which led me to use only IfcRelAdheresToElement in this IFC file.

**Comment by falken10vdl** [comment]
This looks like a discussion. @fg55b let us now if you wnat to reopen as issue. Otherwise lets continue it as discussion. Thanks!
